components/style/properties/properties.mako.rs

```diff
@@ -577,6 +573,12 @@ impl ShorthandId {
             None => return None
         };
 
+        // Alias shorthand properties like -moz-transform should not be
+        // serialized as shothand. They will be serialized as londhand.
+        if self.flags().contains(ALIAS_PROPERTY) {
+            return None;
```

Comment:
Note that you still want it to work if someone asks for `-moz-transform` to be serialized. I suspect this is the cause of most of the test failures.